.github/labeler.yml

```diff
@@ -0,0 +1,8 @@
+ci-cd:
```

Comment:
`isso/js` (but not `isso/js/admin.js`, `isso/css` -> `client`
`isso/**/*.py` -> server
(`isso/css/admin.css`, `isso/js/admin.js`, `isso/templates/admin.html` -> admin)